### PR TITLE
Airgap refactor

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR refactors the airgap install steps for VMware.

## Major Changes

- [VerteX VMware Airgap Instructions](https://deploy-preview-2003--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/airgap/vmware-vsphere-airgap-instructions)
- [Palette VMware Airgap Instructions](https://deploy-preview-2003--docs-spectrocloud.netlify.app/enterprise-version/install-palette/airgap/vmware-vsphere-airgap-instructions)

## Minor Changes

- [VerteX Checklist](https://deploy-preview-2003--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/airgap/checklist) 
- [Palette Checklist](https://deploy-preview-2003--docs-spectrocloud.netlify.app/enterprise-version/install-palette/airgap/checklist)

- [MetalLB Airgap Guidance](https://deploy-preview-2003--docs-spectrocloud.netlify.app/integrations/metallb#airgap-palette-and-vertex)

- [Prometheus Grafana Airgap Guidance](https://deploy-preview-2003--docs-spectrocloud.netlify.app/integrations/prometheus-operator#airgap-palette-and-vertex)

## Review Changes

💻 [Preview URL](https://deploy-preview-2003--docs-spectrocloud.netlify.app)

🎫 [DOC-973](https://spectrocloud.atlassian.net/browse/DOC-973)
🎫 [DOC-1043](https://spectrocloud.atlassian.net/browse/DOC-1043)
🎫 [DOC-1044](https://spectrocloud.atlassian.net/browse/DOC-1044)

### CODE REVIEW

File: docs/docs-content/enterprise-version/install-palette/airgap/checklist.md [hunk 1]
```diff
@@ -24,66 +24,44 @@ installation.
...
 
-- [ ] Downloaded the airgap setup binary from the support team.
+- [ ] Download the airgap setup binary from the support team.
 
-- [ ] Created a private repository with the name `spectro-packs` in your OCI registry. You can use a different name if
+- [ ] Create a private repository with the name `spectro-packs` in your OCI registry. You can use a different name if
       you prefer.
 
-- [ ] Created a public repository with the name `spectro-images` in your OCI registry. You can use a different name if
+- [ ] Create a public repository with the name `spectro-images` in your OCI registry. You can use a different name if
```
Comment (tiwarisumit3) [inline docs/docs-content/enterprise-version/install-palette/airgap/checklist.md:32]: This all steps are not required if we use OVA. Lets speak about that oo

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 2]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
 
 - [ ] `zip` is installed and available.
 
-- [ ] Downloaded the airgap setup binary from the support team.
+- [ ] Download the airgap setup binary from the support team.
 
-- [ ] Created a private repository with the name `spectro-packs` in your OCI registry. You can use a different name if
+- [ ] Create a private repository with the name `spectro-packs` in your OCI registry. You can use a different name if
```
Comment (tiwarisumit3) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:29]: This is required if you setup your own OVA instead of Palette's one

**Review by tiwarisumit3 — APPROVED** [review]

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 3]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
...
-- [ ] Authenticated with your OCI registry and acquired credentials to both repositories.
+- [ ] Authenticate with your OCI registry and acquired credentials to both repositories.
 
 - [ ] Download the Certificate Authority (CA) certificate from your OCI registry.
 
 - [ ] Set the required environment variables for the airgap setup binary. The values are different depending on what
       type of OCI registry you use.
 
-- [ ] Started the airgap setup binary and verified the setup completed successfully.
+- [ ] Start the airgap setup binary and verified the setup completed successfully.
```
Comment (yuliiiah) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:42]:
```suggestion
- [ ] Start the airgap setup binary and verify the setup is completed successfully.
```

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 4]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
...
 
-- [ ] Reviewed the list of pack binaries to download and upload to your OCI registry.
+- [ ] Review the list of pack binaries to download and upload to your OCI registry.
 
-- [ ] Extracted the manifest content from the airgap setup binary to an HTTP file server.
+- [ ] Extract the manifest content from the airgap setup binary to an HTTP file server.
 
-- [ ] Ensured the manifest content is hosted on an HTTP file server that is accessible from the environment you are
+- [ ] Ensure the manifest content is hosted on an HTTP file server that is accessible from the environment you are
       installing VerteX.
```
Comment (yuliiiah) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:49]:
```suggestion
- [ ] Ensure the manifest content is hosted on an HTTP file server accessible from the environment to which you are installing VerteX.
```

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 5]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
...
 </TabItem>
 
 <TabItem label="VMware vSphere" value="vsphere">
 
-- [ ] `oras` CLI v1.0.0 is installed and available.
-
-- [ ] `aws` CLI v2 or greater CLI is installed and available.
-
-- [ ] `zip` is installed and available.
+- [ ] Create a vSphere VM and Template folder with the name `spectro-templates`.
```
Comment (yuliiiah) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:55]:
```suggestion
- [ ] Create a vSphere VM and Template folder named `spectro-templates`.
```

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 6]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
...
 
-- [ ] Downloaded the airgap setup binary from the support team.
-
-- [ ] Created a vSphere VM and Template folder with the name `spectro-templates`.
-
-- [ ] Imported the Operating System and Kubernetes distribution OVA required for the install and placed the OVA in the
+- [ ] Import the Operating System and Kubernetes distribution OVA required for the install and placed the OVA in the
       `spectro-templates` folder.
```
Comment (yuliiiah) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:58]:
```suggestion
- [ ] Import the Operating System and Kubernetes distribution OVA required for the installation and place the OVA in the `spectro-templates` folder.
```

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 7]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
...
-
-- [ ] Download the Certificate Authority (CA) certificate from your OCI registry.
-
-- [ ] Set the required environment variables for the airgap setup binary. Refer to step 8 in the
-      [Preparation](#preperation) section for information.
-
-- [ ] Started the airgap setup binary and verified the setup completed successfully
-
-- [ ] Reviewed the list of pack binaries to download and upload to your OCI registry.
+- [ ] Append an `r_` prefix and remove the `.ova` suffix to the OVA name after the import.
```
Comment (yuliiiah) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:60]:
```suggestion
- [ ] Append an `r_` prefix and remove the `.ova` suffix from the OVA name after the import.
```

File: docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md [hunk 8]
```diff
@@ -24,70 +24,44 @@ VerteX installation.
...
 
-- [ ] Extracted the manifest content from the airgap setup binary to an HTTP file server.
+- [ ] Start the airgap setup binary and verified the setup completed successfully.
```
Comment (yuliiiah) [inline docs/docs-content/vertex/install-palette-vertex/airgap/checklist.md:62]:
```suggestion
- [ ] Start the airgap setup binary and verify the setup is completed successfully.
```

**Review by yuliiiah — APPROVED** [review]
It's a massive PR, so there are a lot of comments. Most of them are copy-related, so feel free to accept or decline as you see fit. Otherwise, what an insane section to refactor! Congrats on surviving this :)

File: docs/docs-content/enterprise-version/install-palette/airgap/checklist.md [hunk 9]
```diff
@@ -24,66 +24,44 @@ installation.
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/enterprise-version/install-palette/airgap/checklist.md:24]: I think mentioning is the better approach. It's a reference page the field likes to review with the customer.

**Pull request merged**